test/upgrade-e2e/post_upgrade_test.go

```diff
@@ -109,7 +108,7 @@ func TestClusterExtensionAfterOLMUpgrade(t *testing.T) {
 		assert.Contains(ct, cond.Message, "Installed bundle")
 		assert.Equal(ct, ocv1.BundleMetadata{Name: "test-operator.1.0.1", Version: "1.0.1"}, clusterExtension.Status.Install.Bundle)
 		assert.NotEqual(ct, previousVersion, clusterExtension.Status.Install.Bundle.Version)
-	}, time.Minute, time.Second)
+	}, 3*time.Minute, time.Second)
```

Comment:
At a glance, this seems like an unrelated change. Are you noticing that e2e works fine on `main` but needs this timeout bump on this branch?

Reply:
I will push it outside.
Yes, I notice that sometimes it flakes in the action / GitHub